migrations/1575287807413-remove_create_permissions_thuringia.js

```diff
@@ -0,0 +1,122 @@
+// eslint-disable-next-line no-unused-vars
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)